services/net/transcription/appsettings.json

```diff
@@ -20,7 +20,8 @@
     "AzureRegion": "canadacentral",
     "AzureCognitiveServicesKey": "",
     "ConvertToAudio": ["mp4", "mov", "m4a"],
-    "SendEmailOnFailure": true
+    "SendEmailOnFailure": true,
+    "NoticeEmailTo": "${Service__NoticeEmailTo}"
```

Comment:
Why set these to invalid values?

Reply:
these values are from .env file locally, they are valid, I think.